harness/src/network.rs

```diff
@@ -84,23 +102,13 @@ impl Network {
                     let conf_state = ConfState::from((peer_addrs.clone(), vec![]));
                     let store = MemStorage::new_with_conf_state(conf_state);
                     nstorage.insert(id, store.clone());
-                    let config = Config {
-                        id,
-                        election_tick: 10,
-                        heartbeat_tick: 1,
-                        max_size_per_msg: NO_LIMIT,
-                        max_inflight_msgs: 256,
-                        pre_vote,
-                        tag: format!("{}", id),
-                        ..Default::default()
-                    };
+                    let mut config = config.clone();
+                    config.id = id;
+                    config.tag = format!("{}", id);
                     let r = Raft::new(&config, store).unwrap().into();
                     npeers.insert(id, r);
                 }
-                Some(mut p) => {
-                    p.initial(id, &peer_addrs);
```

Comment:
Why initial is unnecessary?

Reply:
Because all need tests which need `initial` is fixed. I think `initial` is a not a good logic because it does additional changes to the peers, which should be already set correctly when the peer is created by `Raft::new`.